config.json

```diff
@@ -18,6 +18,16 @@
         "text_formatting"
       ]
     },
+    {
+      "slug": "reverse-string",
+      "uuid": "f195282c-c7a3-4a85-bbc1-d4e85e62d54c",
+      "core": false,
+      "unlocked_by": "hello-world",
```

Comment:
How about after two-fer?

Reply:
Sounds good to me, I can make the update after `two-fer` is merged.